src/components/cluster/Cluster.tsx

```diff
@@ -788,7 +788,7 @@ function Environment({
                     { error: 'Environment name is required', regex: /^.*$/ },
                     { error: "Use only lowercase alphanumeric characters or '-'", regex: /^[a-z0-9-]+$/ },
                     { error: "Cannot start/end with '-'", regex: /^(?![-]).*[^-]$/ },
-                    { error: 'Minimum 3 and Maximum 16 characters required', regex: /^.{3,16}$/ },
+                    { error: 'Minimum 1 and Maximum 16 characters required', regex: /^.{1,16}$/ },
```

Comment:
Is this expected? We have the rule for min 3, right?

Reply:
Yes, Utkarsh & Prakarsh decide that namespace can be a minimum 1 letter

Reply:
Okay